Calling free() requires including "stdlib.h". Failure to do so causes:
	../src/physfs.c:312:2: warning: incompatible implicit declaration of built-in function ‘free’

--- src/physfs.c	2012-08-30 02:19:29.136840522 -0500
+++ src/physfs.c.new	2012-08-30 02:35:51.483368621 -0500
@@ -18,6 +18,7 @@
     Nicolas Casalini "DarkGod"
     darkgod@te4.org
 */
+#include <stdlib.h>
 #include <string.h>
 #include "lua.h"
 #include "lauxlib.h"
